src/actions/commentActions/index.js

```diff
@@ -0,0 +1,42 @@
+const { BASE_URL } = process.env;
+
+export const getComments = slug => dispatch => fetch(`${BASE_URL}/${slug}/comments/`, {
+  method: "GET",
+  headers: {
+    "content-type": "application/json",
+  },
+})
+  .then(res => res.json())
+  .then((data) => {
+    dispatch({
+      type: "GET_COMMENTS",
+      payload: data.comments,
+    });
+  })
+
+  .catch(err => dispatch({
+    type: "GET_COMMENTS",
```

Comment:
The type here should probably be a get_comments_failure

Reply:
Thank you. It is done